tests/Unit/Symfony/AwsPackagesProviderTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AsyncAws\Test\Unit\Symfony;
+
+use AsyncAws\Symfony\Bundle\DependencyInjection\AwsPackagesProvider;
+use AsyncAws\Test\ServiceProvider;
+use PHPUnit\Framework\TestCase;
+
+class AwsPackagesProviderTest extends TestCase
+{
+    public function testAllPackagesExists()
+    {
+        $packages = AwsPackagesProvider::getServiceNames();
+
+        foreach (ServiceProvider::getAwsServiceNames() as $service) {
+            $name = lcfirst($service);
```

Comment:
This transformation here depends on what we decide on here: https://github.com/async-aws/aws/pull/192#discussion_r382948229